Packs/FortiSIEM/IncidentFields/incidentfield-Incident_Severity.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
@OfekMQmasters Please ad a prefix to id, name and cliname.

Reply:
Done.